diff_drive_controller/src/odometry.cpp

```diff
@@ -86,6 +86,30 @@ bool Odometry::update(double left_pos, double right_pos, const rclcpp::Time & ti
   return true;
 }
 
+bool Odometry::updateFromVelocity(double left_vel, double right_vel, const rclcpp::Time & time)
```

Comment:
line #70-86 are basically the same as this function. I think that for now we could call `updateFromVelocity` in `update()` until we discover we need some extra special implementation in either position or velocity feedback mode.

Reply:
thanks!